integrations/va-auth0/gitbook-manifest.yaml

```diff
@@ -0,0 +1,28 @@
+name: VA-Auth0
+title: VA Auth0
+icon: ./assets/icon.png
+description: Visitor Authentication with Auth0
+visibility: public #unlisted
+script: ./src/index.tsx
+scopes:
+    - space:metadata:read
+    - space:visitor:auth
+    - space:content:read
+organization: w45sBUBc1JWE1ktYkQfI #d8f63b60-89ae-11e7-8574-5927d48c4877
+summary: |
+    # Overview
+    Visitor Authentication allows you to publish content behind an authentication wall so that your content is only accessible to selected people.
+
+    This integration lets you control access to your published content as determined by Auth0.
+
+    # Configure
+    Install this integration on a space and then populate the configuration screen with the details of your Auth0 application and Auth0 instance
+    Then publish the space with Visitor Authentication, choose this integration as the backend, and hit save.
+
+    Your space is now published with Visitor Authentication using Auth0.
+
```

Comment:
I'd suggest we confirm with marketing once and get their opinion on the copy

Reply:
We should also get `previewImages` from them imo

Reply:
I've asked Marketing for their opinion and preview images, but I think we can add previewImages to the integration later and get this merged.

Reply:
Marketing suggested changes to the copy, which I have now made.